upup/models/cloudup/resources/addons/authentication.aws/k8s-1.12.yaml.template

```diff
@@ -46,11 +46,11 @@ spec:
 
         resources:
           requests:
-            memory: 20Mi
-            cpu: 10m
+            memory: {{ or .Authentication.Aws.MemoryRequest "20Mi" }}
+            cpu: {{ or .Authentication.Aws.CPURequest "10m" }}
           limits:
-            memory: 20Mi
-            cpu: 100m
+            memory: {{ or .Authentication.Aws.MemoryLimit "20Mi" }}
+            cpu: {{ or .Authentication.Aws.CPULimit "10m" }}
```

Comment:
Did you mean to change from the previous value of 100m?

Reply:
Nope, that was a typo. Thanks for catching it.